test/e2e/fixtures/e2e_suite.go

```diff
@@ -74,9 +76,27 @@ func (s *E2ESuite) BeforeTest(string, string) {
 	s.DeleteResources()
 }
 
+func (s *E2ESuite) AfterTest(suiteName, testName string) {
+	if s.T().Failed() { // by default, we don't get good logging at test end
+		println("=== FAIL  " + suiteName + "/" + testName)
```

Comment:
Searching for `FAIL:` in the logs of a failed build is a great way to find failed tests (while `FAIL` returns false positives). Can we add the colon to keep this true?
```suggestion
		println("=== FAIL:  " + suiteName + "/" + testName)
```